packages/edit-post/src/classic.scss

```diff
@@ -11,7 +11,7 @@
 // Themes with theme.json can control this themselves.
 // For full-wide blocks, we compensate for the base padding.
 // These margins should match the padding value above.
-html :where(.editor-styles-wrapper) {
+:where(html.editor-styles-wrapper) {
```

Comment:
Test according to original issue:

- https://github.com/WordPress/gutenberg/issues/35884